islandora_basic_collection.module

```diff
@@ -1011,6 +1022,17 @@ function islandora_basic_collection_block_configure($delta = '') {
       '#description' => t('Placeholder to output, to be replaced by phrase populated by AJAX.'),
       '#default_value' => variable_get('islandora_basic_collection_object_count_listing_placeholder', 'Counting items in collections...'),
     );
+    $block['islandora_basic_collection_count_method'] = array(
+      '#type' => 'select',
+      '#title' => 'Collection count query method',
+      '#options' => array(
+        'SPARQL' => t('SPARQL'),
+        'Solr' => t('Solr'),
+      ),
+      '#default_value' => variable_get('islandora_basic_collection_count_method', 'SPARQL'),
+      '#description' => t('Select method for querying the number of objects and collections. Solr is recommended for larger repositories. Respects XACML.'),
```

Comment:
Confirmed it won't show inactive objects. I would call that expected behaviour. That's a lot of information to put in the description; you think it's necessary?

Reply:
Its the only place where I feel this needs to be said since SPARQL is fetching everything? To be honest I'm fine with how it is... also, are you documenting this option, who is even in charge of the next release???

Reply:
SPARQL version also does not show inactive objects.

> also, are you documenting this option, who is even in charge of the next release???

Documenting: not sure what you mean?

Who is in charge: ha, we don't know yet. There are so few of us active committers for Islandora 7 anymore. :/